_pages/presentation_materials.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: "Apache Beam Presentation Materials"
+permalink: /presentation-materials/
+---
+
+Are you interested in learning more about Apache Beam or giving a talk about Apache Beam? Excellent! The Apache Beam community has created this collection of materials to help you get started. The community periodically reviews these materials to ensure they are accurate and up-to-date.
+
+<div class="panel panel-default">
+  <div class="panel-body">
+    <iframe src="https://drive.google.com/embeddedfolderview?id=0B-IhJZh9Ab52a3JLVXFWMDltcHM#list" width="700" height="300" frameborder="0"></iframe>
+  </div>
+</div>
+
+### Using these materials
+These materials are free to use! If you do use these materials, please remember to give credit to the authors of the original materials. These materials include speaker notes and you can view them by clicking **`View`** and selecting **`Show speaker notes`**. A set of presentations are have `[Long]` or `[Short]` in their name. Some topics, such as the *Apache Beam Model*, have two different presentations (a long and short version) depending on your needs and time constraints.
+
+If you use these materials and questions arise:
+
+  * If anyone asks questions to which you are unsure of the answer, please do not hesitate to contact or direct the question to the [`user@beam.incubator.apache.org`](mailto:user@beam.incubator.apache.org) mailing list. The Apache Beam community is here to help.
+  * The Beam community is also interested in knowing *common* questions which are asked. If you have time, please send some feedback to the [`user@beam.incubator.apache.org`](mailto:user@beam.incubator.apache.org) mailing list with questions which arose so we can help find ways to address these questions in the project documentation and these presentations.
+
+## Contributing new content
+The Apache Beam community is always looking for new core content and contributions! Specifically, the community is focused on developing new content in these areas:
```

Comment:
Not sure about specifically listing additional content in the main page. This is fine for the blog, but we shouldn't track outstanding work on the site. (We have JIRA for that.)

So, if this needs tracking, we should have a JIRA, with appropriate labeling. Ok to link to the JIRA search, if needed. If it doesn't need tracking, I'd just be vague and say that we are always looking for new content. I'd probably do the latter.